hw/ip/alert_handler/dv/alert_handler_generic_sim_cfg.hjson

```diff
@@ -15,14 +15,11 @@
   tool: vcs
 
   // Fusesoc core file used for building the file list.
-  fusesoc_core: lowrisc:dv:alert_handler_sim:0.1
+  fusesoc_core: lowrisc:dv:alert_handler_generic_sim:0.1
 
   // Testplan hjson file.
   testplan: "{proj_root}/hw/ip/alert_handler/data/alert_handler_testplan.hjson"
 
-  // RAL spec - used to generate the RAL model.
-  ral_spec: "{proj_root}/hw/ip/alert_handler/data/alert_handler.hjson"
-
```

Comment:
please correct if I was wrong, I do not think it is needed since we have the generate flow for ral in fusesoc.